build.sbt

```diff
@@ -59,6 +63,7 @@ val thisBuildSettings = inThisBuild(
     crossScalaVersions := Seq("2.12.10"), //, "2.13.0"),
     licenses := Seq(("MIT", url("https://opensource.org/licenses/MIT"))),
     resolvers += Resolver.bintrayRepo("evolutiongaming", "maven"),
-    releaseCrossBuild := true
+    releaseCrossBuild := true,
+    skip in publish := true
```

Comment:
I'd rather recommend to use `skip in publish := false` when needed, and not other way around like it is now.